compiler/tests-ocaml/lib-unix/isatty/dune

```diff
@@ -0,0 +1,7 @@
+(tests
+ (names isatty_std)
+ ; isatty_tty does not work on the CI since we are not running in a tty there
```

Comment:
Can we keep it when running tests locally ? Is there any magic `(enabled_if <XXX>)` we can use ?

Reply:
Pushed a fix